src/calculator.js

```diff
@@ -26,7 +26,7 @@ exports.subtract = (x, y) => {
   return x - y;
 };
 
-exports.multiply = (x, y) => {
+exports.multiply = (x,y) => {
```

Comment:
A space is required after &#39;,&#39;.

###### See http://eslint.org/docs/rules/comma-spacing for details about this rule